docs/source/usage.rst

```diff
@@ -34,28 +34,15 @@ heavyai
 If you have an NVIDIA GPU in the same machine where your heavyai code will be running, you'll want to `install
 cudf`_ as well to return results sets into GPU memory as a cudf GPU DataFrame:
 
-cudf via conda
-**************
+cudf
+****
 
-.. code-block:: console
-
-   # CUDA 9.2
-   conda install -c nvidia -c rapidsai -c numba -c conda-forge -c defaults cudf
-
-   # CUDA 10.0
-   conda install -c nvidia/label/cuda10.0 -c rapidsai/label/cuda10.0 -c numba \
-       -c conda-forge -c defaults cudf
-
-cudf via PyPI/pip
-*****************
+cudf is only available on conda. Hence, the use of a conda environment is
+required.
```

Comment:
nit: strictly speaking, cudf can be built from source to a non-conda environment as well. Hence:
```suggestion
The pre-built cudf is only available on conda. Hence, the use of a conda environment is required.
```